main.go

```diff
@@ -1,19 +1,26 @@
 package main
 
 import (
-	"fmt"
+	"budget-helper/database"
+	"log"
 	"net/http"
 )
 
-const PORT = ":8080"
+const port = ":8080"
 
 func main() {
 	// Initialization steps:
 	// - [x] Register routes in the http.ServeMux called "router"
-	// - [ ] Connect to database
-	// - [x] Listen for connections on PORT
+	// - [x] Connect to database
+	// - [x] Listen for connections on port
 	router := initRouter()
 
-	fmt.Printf("Server started on PORT %v...\n", PORT)
-	http.ListenAndServe(PORT, router)
+	db := database.NewDatabase()
+	defer db.Close()
+
+	log.Printf("Server started on PORT %v...\n", port)
+	err := http.ListenAndServe(port, router)
```

Comment:
```
log.Fatal(http.ListenAndServe(port, router))
```

`ListenAndServe` only terminates if something goes wrong, and returns an error; this means you can wrap it directly.